lib/valkyrie/persistence.rb

```diff
@@ -37,5 +37,7 @@ class ObjectNotFoundError < StandardError
     end
     class UnsupportedDatatype < StandardError
     end
+
+    OPTIMISTIC_LOCK_ATTRIBUTE = :optimistic_lock_token
```

Comment:
I wonder if we should namespace all of these kinds of things? Something like `Valkyrie::Persistence::Attributes::OPTIMISTICLOCK`?

Reply:
@tpendragon would this come up in a grep search?

Reply:
`OPTIMISITIC_LOCK`?